src/components/slider/partial-styles/_readonly.scss

```diff
@@ -11,16 +11,16 @@
 
         .mdc-slider__pin,
         .mdc-slider__track {
-            background-color: var(
-                --lime-primary-color,
-                mdc-variables.$mdc-theme-primary
-            );
+            // background-color: var(
```

Comment:
We have 2 issues here: 
1. The knob is not on the slider bar. 
2. We should remove the shadow from `.mdc-slider__thumb`. 

We don't see the value inside the knob, this might not be an issue but I should mention it here since it is a different feature. The slider value can be seen on top of the knob when draging the knob

<img width="441" alt="Screenshot 2021-07-15 at 09 59 04" src="https://user-images.githubusercontent.com/22968497/125753372-b4f40133-d2a0-43c3-a8b7-742cddc0f529.png">